src/include/target/AION_2400_T-Pro_TX.h

```diff
@@ -0,0 +1,30 @@
+#define DEVICE_NAME "AION T-Pro TX"
+
+// Any device features
+#define USE_SX1280_DCDC
+
+// GPIO pin definitions
+#define GPIO_PIN_NSS            5
+#define GPIO_PIN_BUSY           21
+#define GPIO_PIN_DIO1           4
+#define GPIO_PIN_DIO2           22
+#define GPIO_PIN_MOSI           23
+#define GPIO_PIN_MISO           19
+#define GPIO_PIN_SCK            18
+#define GPIO_PIN_RST            14
+#define GPIO_PIN_RX_ENABLE      27
+#define GPIO_PIN_TX_ENABLE      26
+#define GPIO_PIN_PA_ENABLE      25
+#define GPIO_PIN_RCSIGNAL_RX    3
+#define GPIO_PIN_RCSIGNAL_TX    1
+
+// Backpack pins
+#define GPIO_PIN_DEBUG_RX       13 // 16 in production hopefully!
+#define GPIO_PIN_DEBUG_TX       12 // 17 in production hopefully!
+
```

Comment:
Neither GPIO_PIN_DEBUG pins are broken out to pads.  So I don't think it would hurt to change these to 16 and 17 as they should be.

Reply:
It seems that the program will not run after deletion

Reply:
Currently just remove the comments at the end of the line, not the whole line.

Reply:
This file would also be better named `Jumper_AION_2400_T-Pro_TX.h`